src/debugger/Debugger.cc

```diff
@@ -1008,11 +1018,14 @@ string Debugger::Cmd::help(std::span<const TclObject> tokens) const
 		"  Note that openMSX comes with a 'disasm' Tcl script that is much "
 		"more convenient to use than this subcommand.";
 	auto disasmBlobHelp =
-		"debug disasm_blob <value> <addr>\n"
+		"debug disasm_blob <value> <addr> [<function>]\n"
 		"  This is a more generic version of the disasm subcommand, but it "
 		"works on a Tcl binary string (see Tcl manual) to disassemble a "
 		"single instruction. The given address is used when relative "
-		"address to jump to is necessary.\n";
+		"address to jump to is necessary. The optional fuction will be "
+		"called with an address as parameter and may return a symbol name "
+		"that replaces that address if a symbol that matches the address "
+		"is found.\n";
```

Comment:
Detail: this help text says "... and _may_ return a symbol name ...". But in the current implementation the function _must_ return a string. And that string is always used, also if it is e.g. an empty string.
An alternative could be to fallback to `appendAddrAsHex()` if the function returns an empty string.
Either alternative is fine for me, pick which you like best: tweak the help text or change the implementation.

Can you also test what happens when the function returns an error instead of a string (e.g. as a test intentionally write an error in your function). What should happen is that the `debug disasm_blob` command fails with that error. E.g. it should not crash openMSX. I think the current implementation might already be fine, but we should make sure.

Can you also document `disasm_blob` in `doc/manual/commands.html`. Search for `debug disasm`, a similar single line documentation should be sufficient (no need for any details, the help text you added in openMSX itself already has the details).

Reply:
> Can you also test what happens when the function returns an error instead of a string (e.g. as a test intentionally write an error in your function). What should happen is that the `debug disasm_blob` command fails with that error. E.g. it should not crash openMSX. I think the current implementation might already be fine, but we should make sure.

It seems to return a TclException.
![image](https://github.com/user-attachments/assets/33577aef-4b18-48a7-840d-44ad052b4c51)